test/integration/common/StudioActions.kt

```diff
@@ -235,15 +223,22 @@ object Utils {
         }
     }
 
-    private fun fileNameToString(fileName: String): String {
-        return Files.readAllLines(Paths.get(fileName), StandardCharsets.UTF_8).filter { line -> !line.startsWith('#') }
-            .joinToString("")
+    private fun readQueryFileToString(queryFileName: String): String {
+        return Files.readAllLines(Paths.get(queryFileName), StandardCharsets.UTF_8)
+            .joinToString("\n")
     }
-}
 
-object Delays {
-    const val RECOMPOSE = 500
-    const val FILE_IO = 750
-    const val NETWORK_IO = 1_500
-    const val CONNECT_SERVER = 2_500
-}
+    object Errors {
+        private const val CONNECT_TYPEDB = "Failed to connect to TypeDB."
+        private const val CREATE_DATABASE = "Failed to create the database."
+        private const val DATA_WRITE = "Failed to write the data."
+        private const val SCHEMA_WRITE = "Failed to write the schema."
+    }
+
+    object Delays {
+        const val RECOMPOSE = 500
+        const val FILE_IO = 750
+        const val NETWORK_IO = 1_500
+        const val CONNECT_SERVER = 2_500
+    }
```

Comment:
Delays as a class was looking pretty thin, so now we have `StudioActions.Delays` - these are delays for actions taken within Studio, so this naming makes sense.